spec/ParseLiveQuery.spec.js

```
@@ -1308,4 +1308,59 @@ describe('ParseLiveQuery', function () {
+    
+    const subscription = await query.subscribe();
+    subscription.on('create', () => {});
+
+    // Do not need any expect block, just make sure that the server doesn't crash or throw error
+    const object1 = new TestObject();
+    object1.set('childs', [child3]);
+    await object1.save();
+  });
+
+
+  fit('Live query should work if we set equalTo(someKey,someParseObject) and new Parse object is created but someKey = null', async () => {
```

Biome (1.9.4)

[error] Don't focus the test.

The 'only' method is often used for debugging or during implementation. It should be removed before deploying to production.
Consider removing 'only' to ensure all tests are executed.
Unsafe fix: Remove focus from test.


(lint/suspicious/noFocusedTests)